src/components/Article.jsx

```diff
@@ -32,8 +54,10 @@ function Article() {
             <p>Article by: {article.author}</p>
             <p>{article.body}</p>
             <p>Topic: {article.topic}</p>
-            <input placeholder="Write a comment"/>
-            <p>Comment Count: {article.comment_count}</p>
+            <p>Votes: {article.votes}</p>
+            <button onClick={() => handleVote(1)}>+1</button>
+            <button onClick={() => handleVote(-1)}>-1</button>
+            <p>{voteError}</p>
```

Comment:
You would probably want to provide a more user-friendly error message, maybe something like "whoops, unable to process that vote" or something less intimidating. 
Remember, the user is _probably_ not a developer.